resources/assets/js/annotations/components/annotationCanvas/translateInteraction.vue

```diff
@@ -30,21 +30,27 @@ export default {
                 translateInteraction.setActive(translating);
             }
         },
+        canModify(state){
+            if(state){
+                Keyboard.on('m', this.toggleTranslating, 0, this.listenerSet);
+            }
+            else{
+                translateInteraction.setActive(false);
+                Keyboard.off('m', this.toggleTranslating, 0, this.listenerSet);
+            }
+        }
     },
     mounted() {
         // Initialize the translate interaction here because we have to wait for
         // the non-reactive properties of annotationCanvas to be initialized.
-        if (this.canModify) {
             translateInteraction = new TranslateInteraction({
```

Comment:
Please fix the indentation.

Reply:
For the next time, could you use the code from my branch, when you highlight something here? Sometimes it's quiet confusing to understand what you really mean, if lines of code are highlighted which don't exist on my branch any more.

Reply:
You mean the links in my [main comment](https://github.com/biigle/core/pull/611#pullrequestreview-1510599393)? I'll try to remember this next time!

If you mean *this* comment, the highlighted line should be in your branch. :thinking:

Reply:
No, I mean this comment. The if-statement in line 37 does not exist on my branch [img-fail-view.](https://github.com/biigle/core/blob/49c058e090eea1a886e18c1c3f9281a427a26b7c/resources/assets/js/annotations/components/annotationCanvas/translateInteraction.vue#L43-L54)

Reply:
Ah I see. Line 37 was deleted in your branch, as is shown above. My comment refers to line 38/46 which must be unindented because the `if` is no longer there.